src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -205,7 +210,92 @@ public Movie GetMovieInfo(int TmdbId, Profile profile = null)
                     movie.Studio = resource.production_companies[0].name;
                 }
             }
-
+            movie.AllFlicksUrl = null;
+            string enableAllFlicks = _settingsService.EnableAllFlicks;
+            if ((enableAllFlicks!="disabled")&&((enableAllFlicks=="simple")||(enableAllFlicks=="complex")))
+            {
+                string countryCode = _settingsService.NetflixCountryCode;
+                string referer;
+                switch (countryCode)
+                {
+                    case "ca":
+                        {
+                            referer = "https://www.allflicks.net/canada/";
+                            break;
+                        }
+                    case "us":
+                    default:
+                        {
+                            referer = "https://www.allflicks.net/";
+                            break;
+                        }
+                }
+                if (enableAllFlicks=="simple")
+                {
+                    using (WebClient client = new WebClient())
```

Comment:
I briefly looked into this. i havent been able to get it to work .. yet.. without simply using WebCLient inside: NzbDrone.Common/Http/HttpClient.cs